### ISSUE TITLE
[Feature Request] New Characters Panel instead of Persona management menu (with concept)

### ISSUE THREAD

**Issue by ahmed-elj** [SillyTavern/SillyTavern#1093]
Hey! I am back with another crazy idea! as always.
**Have you searched for similar [requests](https://github.com/SillyTavern/SillyTavern/issues?q=)?**
Yes

Description:
So the persona management tab is barely used, and it's taking a good portion of the UI, on the contrary the character management sidebar is over stuffed with UI elements (favorite button, group button, create character, search bar [which is very tiny!], etc....).

I am suggestion a major switch between character sidebar and persona management floating menu, so like this quick concept i made.

this is the new Characters panel™️, the prior persona management menu:
![plan characters](https://github.com/SillyTavern/SillyTavern/assets/31710770/1c4f6311-ab3e-4207-a9db-e9532fc76372)

And the new place for the persona management is shared with the current character sidebar:
![new side bar](https://github.com/SillyTavern/SillyTavern/assets/31710770/7b95b962-7470-46c6-a1e9-3d195e507fd9)

by clicking on the name of your persona, in this case "Jake", you switch from the current character options to the persona management, 

you can differentiate the current active menu by the lit up name (either the persona name or the character name).

So in conclusion, we can swap places of characters list from the sidebar to the persona management and rename to character panel.
now the "persona management" moved to share the sidebar with the "character sidebar". (a bit confusing, but the screenshots can explain it).
So by doing this, we can expand the characters list space, and extend the search bar a bit, and more space for future plans.

Feedback is always appreciated, 
Thank you.
